continuous_integration/scripts/run_tests.sh

```diff
@@ -10,8 +10,8 @@ if [[ $PARALLEL == 'true' ]]; then
 fi
 
 if [[ $COVERAGE == 'true' ]]; then
-    echo "coverage run `which py.test` dask --runslow --doctest-modules $XTRATESTARGS"
-    coverage run `which py.test` dask --runslow --doctest-modules $XTRATESTARGS
+    echo "coverage run -m pytest dask --runslow --doctest-modules $XTRATESTARGS"
+    coverage run -m pytest dask --runslow --doctest-modules $XTRATESTARGS
```

Comment:
We moved doctests to a separate GH actions build (xref https://github.com/dask/dask/pull/7238)

```suggestion
    echo "coverage run -m pytest dask --runslow $XTRATESTARGS"
    coverage run -m pytest dask --runslow $XTRATESTARGS
```